route/kafkamdm.go

```diff
@@ -127,13 +127,20 @@ func (r *KafkaMdm) run() {
 	ticker := time.NewTicker(r.flushMaxWait)
 	var client sarama.Client
 	var err error
+	attempts := 0
 
 	for r.producer == nil {
 		client, err = sarama.NewClient(r.brokers, r.saramaCfg)
 		if err == sarama.ErrOutOfBrokers {
 			log.Warnf("kafkaMdm %q: %s", r.key, err)
 			// sleep before trying to connect again.
 			time.Sleep(time.Second)
+			attempts++
+			// fail after 300 attempts
+			if attempts > 300 {
+				log.Fatalf("kafkaMdm %q: no kafka brokers available for more than 5 minutes", r.key)
```

Comment:
i'm not sure if it's a good idea to state `5 minutes` here. i assume you wrote that because we get here after waiting 300 times for 1 sec each. but imagine if f.e. `sarama.NewClient()` takes 10 seconds to time out, then we only get here after almost an hour, so then it would be weird if the msg says `5 minutes`

Reply:
True, I will change that